packages/account/src/Sections/Profile/LanguageSettings/language-settings.tsx

```diff
@@ -9,7 +9,7 @@ import { Formik, FormikHandlers, FormikHelpers, FormikValues } from 'formik';
 import FormFooter from 'Components/form-footer';
 import LanguageRadioButton from 'Components/language-settings';
 
-type TLanguageSettings = {
+export type TLanguageSettings = {
```

Comment:
@yauheni-deriv No need to export the type, You can use `React.ComponentProps` generic instead.

Reply:
@yauheni-deriv Please ignore my comment, The component is using `connect` which messes up the types so we can't use types :(

Reply:
this types are removed and component is refactored with useStores